gridt/src/app/app.component.ts

```diff
@@ -36,6 +40,10 @@ export class AppComponent implements OnInit, OnDestroy{
     private auth: AuthService
   ) {
     this.initializeApp();
+    this.isLoggedIn$=this.router.events.pipe(
```

Comment:
It might be good to leave a comment explaining why it is necessary to do the piping, because that might not be completely clear from just looking at the code.